src/settings-ui/Settings.UI.Library/MeasureToolProperties.cs

```diff
@@ -18,6 +18,7 @@ public MeasureToolProperties()
             DrawFeetOnCross = true;
             PerColorChannelEdgeDetection = false;
             MeasureCrossColor = new StringProperty("#FF4500");
+            DefaultMeasureStyle = new IntProperty(0);
```

Comment:
Can we use the enum name please. No one will know what 0 means without checking it.

Reply:
Oh, I missed that. Thanks!